src/mono/wasm/build/WasmApp.targets

```diff
@@ -99,8 +99,9 @@
 
   <PropertyGroup>
     <WasmDedup Condition="'$(WasmDedup)' == ''">true</WasmDedup>
-    <WasmEnableExceptionHandling Condition="'$(WasmEnableExceptionHandling)' == ''">false</WasmEnableExceptionHandling>
-    <WasmEnableSIMD Condition="'$(WasmEnableSIMD)' == ''">true</WasmEnableSIMD>
+    <!-- Post Wasm MVP features -->
+    <WasmEnableExceptionHandling Condition="'$(WasmEnableExceptionHandling)' == ''">true</WasmEnableExceptionHandling>
+    <WasmEnableSIMD Condition="'$(WasmEnableSIMD)' == ''">$(WasmEnableExceptionHandling)</WasmEnableSIMD>
```

Comment:
curious: why to base the default SIMD value on EH value?

Reply:
> EH is more broadly compatible/deployed than SIMD

only in Safari. Chrome, Firefox and nodejs have SIMD support longer than EH. (https://webassembly.org/roadmap/)

I think the new behavior is a bit confusing. Maybe we could introduce a new property, like `WasmEnableAdvancedFeatures` or `WasmEnableNextGen` to switch the defaults of SIMD, EH and other non 1.1 features. What do you think?